tools/preview/main.js

```diff
@@ -30,116 +35,120 @@ module.exports = {
     '@etchteam/storybook-addon-status',
   ],
   env: {
-    MIGRATED_PACKAGES: componentPkgs.filter(dir => {
-      const pkg = require(resolve(componentsPath, dir, 'package.json'));
-      if (pkg.devDependencies && pkg.devDependencies['@spectrum-css/component-builder-simple']) {
-        return true;
+    MIGRATED_PACKAGES: componentPkgs.filter((dir) => {
+      const pkg = require(resolve(componentsPath, dir, 'package.json'))
+      if (
+        pkg.devDependencies &&
+        pkg.devDependencies['@spectrum-css/component-builder-simple']
+      ) {
+        return true
       }
-      return false;
+      return false
     }),
   },
-  core: {
-    /* Do not send analytics to storybook */
-    disableTelemetry: true,
```

Comment:
https://storybook.js.org/docs/react/configure/telemetry

Reply:
In Storybook 7, there was no provisions for using the core object so I removed it. But yes since we dont want analytics not to be sent I have added disableTelemetry: true and added a core object.